apps/graph/values/values_controller.cpp

```diff
@@ -41,6 +41,18 @@ ValuesController::ValuesController(Responder * parentResponder, InputEventHandle
   m_selectableTableView.setDelegate(this);
 }
 
+Shared::Hideable * ValuesController::hideableCellFromType(HighlightCell * cell, int type) {
+  if (type == k_notEditableValueCellType) {
+    Shared::HideableEvenOddBufferTextCell * myCell = static_cast<Shared::HideableEvenOddBufferTextCell *>(cell);
+    return static_cast<Shared::Hideable *>(myCell);
+  } else if (type == k_editableValueCellType) {
```

Comment:
Remove else if case and replace with:
assert(type == k_editableValueCellType);
Shared::StoreCell * myCell = static_cast<Shared::StoreCell *>(cell);
return static_cast<Shared::Hideable *>(myCell);

?